.babelrc.js

```diff
@@ -16,6 +16,13 @@ if (process.env.NODE_ENV === 'test') {
   plugins.push("dynamic-import-node");
 }
 
+if (process.env.NODE_ENV === 'production') {
```

Comment:
I believe these only removes prop-types from simorgh code, it would be safe to transpile all dependencies and remove prop-types imo, but certainly would be necessary to do it for all psammead dependencies.

Reply:
Yep, looks like the `vendor` bundles are not changed by this. Docs for the plugin claim it includes everything (including `node_modules`) by default - will investigate what is happening